packs/equipment-effects/effect-aurochs-endurance.json

```diff
@@ -0,0 +1,46 @@
+{
+    "_id": "mIZVE5kyzxScpaS5",
+    "img": "icons/consumables/meat/steak-alien-orange.webp",
+    "name": "Effect: Aurochs' Endurance",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Aurochs Jerky]</p>\n<p>You gain a +2 item bonus to saving throws against effects that would fatigue you.</p>"
```

Comment:
We should also AdjustModifier away the fatigued penalty. Can't suppress the condition itself but we can at least do that